### DISCUSSION TITLE
Access to private repo with access token

### DISCUSSION THREAD

**Discussion by MrCarb0n** [gitpython-developers/GitPython#1252] [Q&A] [answered]
Hi, can we access to private repo with github access token? not with ssh key btw.

**Comment by Byron** [comment, marked as answer]
As far as I know [this works using special URLs](https://github.blog/2012-09-21-easier-builds-and-deployments-using-git-over-https-and-oauth/) and thus is outside of GitPython.

You might also be interested in various [token formats](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/about-authentication-to-github#githubs-token-formats).
